zmon-common/src/main/java/org/zalando/zmon/domain/SumMinuteChecks.java

```diff
@@ -0,0 +1,9 @@
+package org.zalando.zmon.domain;
+
+import java.util.List;
+
+public class SumMinuteChecks {
```

Comment:
You use public fields and no getters/setters here, but not in `FrontendBootData`, why? I would follow the standard Java POJO format everytwhere

Reply:
Also, should it be Su**b**MinuteChecks?